python/ray/tests/test_tracing.py

```diff
@@ -0,0 +1,100 @@
+import os
+
+# TODO: use py_test(env = ...) in the build file with bazel 4.0
+os.environ["RAY_TRACING_ENABLED"] = "True"
+
+from opentelemetry import trace  # noqa: E402
+from opentelemetry.sdk.trace import TracerProvider  # noqa: E402
+from opentelemetry.sdk.trace.export import (
+    ConsoleSpanExporter,
+    SimpleExportSpanProcessor,
+)  # noqa: E402
+import pytest  # noqa: E402
+import shutil  # noqa: E402
+import tempfile  # noqa: E402
+from typing import Any  # noqa: E402
+import ray  # noqa: E402
+
+# Create temporary spans folder for trace output.
+spans_dir = tempfile.gettempdir() + "/spans"
+
+
+@pytest.fixture(scope="session")
+def cleanup_dirs():
+    """Cleanup temporary spans_dir folder at beginning and end of test."""
+    if os.path.exists(spans_dir):
+        shutil.rmtree(spans_dir)
+    os.makedirs(spans_dir)
+    yield
+    print("at end of test")
+    if os.path.exists(spans_dir):
+        shutil.rmtree(spans_dir)
+
+
+def _setup_tracing(*args: Any, **kwargs: Any) -> None:
```

Comment:
writes a note saying this is what the users need to do right now, and we should consider do this automatically in the future.